twilight-model/src/util/hex_color.rs

```diff
@@ -0,0 +1,122 @@
+use std::fmt::Formatter;
+use std::fmt::{Display, Result as FmtResult};
+use std::num::ParseIntError;
+use std::str::FromStr;
+
+use serde::de::Visitor;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+pub struct HexColor(pub u8, pub u8, pub u8);
```

Comment:
If the RGB values are `pub`, we should at least document the order, since BGR is a thing as well. The struct should also generally have documentation.